koku/api/report/ocp/serializers.py

```diff
@@ -170,12 +170,16 @@ def validate(self, data):
             (ValidationError): if field inputs are invalid
 
         """
+        if "virtual-machines" not in str(self.context.get("request")):
```

Comment:
What if someone has a query like:
```
http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by[project]=virtual-machines
```

Reply:
group bys are not allowed on this endpoint, but I get your point